hooks/pre-command

```diff
@@ -18,11 +18,54 @@ ecr_exists() {
     --query 'repositories[0].registryId'
 }
 
+get_ecr_arn() {
```

Comment:
Might be able to combine this with ecr_exists, as it would avoid an extra API call and you need the arn anyway. Not a big deal tbh.